infrastructure/init.sh

```diff
@@ -102,7 +106,7 @@ fi
 
 
 # Create EC2 Key-Pair that will be used to ssh into EC2 instances spin up by terraform module
-if aws ec2 create-key-pair --key-name "${project}-key" --tag-specifications "ResourceType=key-pair,Tags=[{Key=Project,Value=${project}},{Key=Owner,Value=${owner_email}},{Key=Creator,Value=${creator_email}}]" --query 'KeyMaterial' --output text > "${project}"-key.pem --region="${aws_region}" ; then
+if aws ec2 create-key-pair --key-name "${namespace}-key" --tag-specifications "ResourceType=key-pair,Tags=[{Key=Project,Value=${project}},{Key=Owner,Value=${owner_email}},{Key=Creator,Value=${creator_email}}]" --query 'KeyMaterial' --output text > "${project}"-key.pem --region="${aws_region}" ; then
```

Comment:
update same command in init-delete.sh as well

Reply:
Updated teh init_delete.sh file